### ISSUE TITLE
Fetching SSM variables from file v2.28.1

### ISSUE THREAD

**Issue by rfoel** [serverless/serverless#9048]
While deploying a service I noticed that fetching SSM variables from a config file stopped working when using the latest release v2.28.1. I understand that there's some deprecation but it shouldn't come until the v3, correct? The output in my lambda is the string from my config file now. 

```
// config.yml
MY_SECRET_KEY: ${ssm:/path/to/my/secret/key~true}
```

```
// serverless.yml
  environment:
    MY_SECRET_KEY: ${file(config.yml):MY_SECRET_KEY}
```

<b>Installed version</b>

```
Framework Core: 2.28.1
Plugin: 4.4.3
SDK: 2.3.2
Components: 3.7.1
```

**Comment by medikoo** [comment]
@rfoel thanks for reporting. It definitely should remain working as it worked, let me check what happened

**Comment by tmanning** [comment]
This still fails in v2.28.2:
```
iam:
  role:
    statements:
      - Effect: Allow
        Action:
          - dynamodb:DescribeStream
        Resource:
          - ${cf.${self:provider.region}:my-project-${self:provider.stage}.MyTableArn}
```
An error occurred: IamRoleLambdaExecution - Resource ${cf.${self:provider.region}:my-project-${self:provider.stage}.MyTableArn} must be in ARN format or "*".

**Comment by medikoo** [comment]
@tmanning Thanks for follow up, will be fixed in next patch

**Comment by medikoo** [comment]
@tmanning fix was just  published with v2.28.3
